e2e-runner/lib/darwin/runner.sh

```
@@ -319,6 +319,13 @@ if (( cleanMachine == 1 )); then
     # remove old podman system connections from user space
     rm -rf ~/.config/containers/podman-connections.json*
     rm -rf ~/.config/containers/podman
+    # Detach any stale DMG volumes from previous runs
+    for vol in /Volumes/${appName}*; do
```

Shellcheck (0.11.0)

[info] Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

(SC2231)